src/Symfony/Component/ErrorHandler/ErrorEnhancer/UndefinedMethodErrorEnhancer.php

```diff
@@ -37,6 +37,10 @@ public function enhance(\Throwable $error): ?\Throwable
         $className = $matches[1];
         $methodName = $matches[2];
 
+        if ('' === $methodName) {
+            return new UndefinedMethodError(sprintf('Attempted to call a method of "%s" without naming it explicitly.', $className), $error);
```

Comment:
I would keep the message below, it's fine
instead of this block, I would change L 42 to:
`if ('' === $methodName || !class_exists($className) || null === $methods = get_class_methods($className)) {`

Reply:
Then the error would be:
> Attempted to call an undefined method named "" of class "Foo".

Not a great fan of the empty quotes, is it really better?

Reply:
I think yes, at least from the pov if maintenance: one less specificity
Also "without naming it explicitly" isn't more clear to me...